packages/kit/src/module/define.ts

```diff
@@ -67,7 +67,25 @@ export function defineNuxtModule<OptionsT extends ModuleOptions> (definition: Mo
     }
 
     // Call setup
-    await definition.setup?.call(null as any, _options, nuxt)
+    const setupStart = process.hrtime()
```

Comment:
Maybe we could use https://nodejs.org/api/process.html#processhrtimebigint here since it's supported in v10?

Reply:
Good idea will refactor! (initially was lazy to import ahah :D)

Reply:
Migrated to performance api